MANIFEST.in

```diff
@@ -0,0 +1,14 @@
+prune .github
+prune .nox
+prune docs
+prune requirements
+recursive-include requirements *
+recursive-include iris_grib *.cml *.grib2 *.json *.nc *.py
+
+exclude .coveragerc
+exclude .flake8
+exclude .gitignore
+exclude .pre-commit-config.yaml
+exclude .readthedocs.yml
+include *.md
```

Comment:
@bjlittle Should we also include *.rst since the iris-grib readme is restructured text?

Reply:
@HGWright Good question, but no.

By default `setuptools` will automagically include any `README` file variation i.e., `.md`, `.rst`, `.txt`.

This particular line is to specifically include the `CONTRIBUTING.md` in the root directory, but in a generic way i.e., mop up any other `.md` files e.g., hopefully we add a `CODE_OF_CONDUCT.md` in the near future 😉 

See [here](https://setuptools.pypa.io/en/latest/deprecated/distutils/sourcedist.html#specifying-the-files-to-distribute) for further details.